config/initializers/time.rb

```diff
@@ -1,7 +1,7 @@
 # Extends time class with convenience methods
 class Time
   def display_date
-    getlocal.strftime("%Y-%m-%d")
+    getlocal.strftime("%m-%d-%Y")
```

Comment:
🕯 I'm going to miss u YYYY-MM-DD, the best date format 🕯 

Can we do slashes here?